tools/icu/icutrim.py

```diff
@@ -159,9 +160,8 @@ def runcmd(tool, cmd, doContinue=False):
     return rc
 
 ## STEP 0 - read in json config
-fi= open(options.filterfile, "rb")
-config=json.load(fi)
-fi.close()
+with io.open(options.filterfile, encoding='utf-8') as fi:
+    config=json.load(fi)
```

Comment:
nit: might as well add the spaces around `=` on this line while we are here

Reply:
@aqrln I agree with you that the python script do not follow the [PEP008 ](https://www.python.org/dev/peps/pep-0008/) and [PEP0020](https://www.python.org/dev/peps/pep-0020/)
But for this we should change more that only theses missing whitespace around equal.

As this feature suggestion is not part of the current bug, I think it would be better to create a merge request for this enhancement.

Reply:
On other projects, I have gotten used to running full __flake8__ only on modified lines:
`git diff master -U0 | flake8 --diff --max-line-length=88 --statistics`

A very gradual migration to psf/black.